frontend/cypress/tests/codeView/codeView_codeHighlighting.cy.js

```diff
@@ -0,0 +1,68 @@
+// Assumes: RepoSense repo from 03/05/2018 to current date
+describe('code highlighting works properly', () => {
+  it('should highlight code when there is a single author', () => {
+    // open the code panel
+    cy.get('.icon-button.fa-code')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-authorship .files', { timeout: 90000 })
+      .should('be.visible');
+
+    cy.get('.hljs-comment').contains('* Represents a Git Author.')
+      .parent() // .line-content
+      .parent() // .code
+      .should('have.css', 'background-color', 'rgb(230, 255, 237)'); // #e6ffed
+  });
+
+  it('should highlight code when multiple authors are merged in a repo group', () => {
+    cy.get('div.mui-select.grouping > select:visible')
+      .select('groupByRepos');
+
+    cy.get('#summary label.merge-group > input:visible')
+      .should('be.visible')
+      .check()
+      .should('be.checked');
+
+    // open the code panel
+    cy.get('.icon-button.fa-code')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-authorship .files', { timeout: 90000 })
+      .should('be.visible');
+
+    cy.get('.hljs-comment').contains('* MUI Colors module') // eugenepeh
+      .parent() // .line-content
+      .parent() // .code
+      .should('have.css', 'background-color')
+      .and('not.eq', 'rgb(255, 255, 255)') // #ffffff
```

Comment:
I think the background color used is always consistently chosen, is it not possible to check for the exact color here instead of checking for not white? 

Doing this would make the test case more robust, ensuring the merged group code is highlighted based on author's color and not all just green.

Reply:
Thanks for pointing that out! I went to look at how the colours are generated (in `c-authorship.vue`) and it turns out that the first 10 colours are consistently chosen, while the 11th onward is randomly generated. I'll amend the code since we're only testing the first two colours.